lib/nrf_modem_lib/nrf91_sockets.c

```diff
@@ -1089,4 +1107,16 @@ NET_DEVICE_OFFLOAD_INIT(nrf91_socket, "nrf91_socket",
 			&nrf91_socket_iface_data, NULL,
 			0, &nrf91_if_api, 1280);
 
-#endif
+#if defined(CONFIG_NRF91_NET_L2)
+/* Bind l2 connectity APIs. */
+static struct net_l2_conn_api nrf91_conn_api = {
+	.init = nrf91_net_l2_init,
+	.connect = nrf91_net_l2_connect,
+	.disconnect = nrf91_net_l2_disconnect,
```

Comment:
`nrf91_net_l2_init` should be renamed `nrf91_l2_conn_init`

`nrf91_net_l2_connect` and `nrf91_net_l2_disconnect` are probably fine, but it'd be fine too if you want to name them `nrf91_l2_conn_connect` and `nrf91_net_l2_conn_disconnect` for consistency

Reply:
`nrf91_net_l2_conn_init`
`nrf91_net_l2_conn_connect`
`nrf91_net_l2_conn_disconnect`

Why not prefix similarily?

Reply:
`nrf91_l2_conn_connect` was a typo, whoops!

I see no reason not to go with the three you suggest